packs/data/pathfinder-bestiary.db/adult-bronze-dragon-spellcaster.json

```
@@ -3391,7 +3391,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:80|traits:arcane,electricity,evocation,damaging-effect]{80-foot line} that deals [[/r 8d12[electricity]]] damage (@Check[type:reflex|dc:33|basic:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:80|traits:arcane,abjuration,incapacitation,mental]{80-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|dc:33] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
+                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:80]{80-foot line} that deals [[/r 8d12[electricity]]] damage (@Check[type:reflex|traits:arcane,electricity,evocation|overrideTraits:true|dc:33|basic:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:80]{80-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true|dc:33] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```

```suggestion
                    "value": "<p>The bronze dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/br 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Lightning</strong> The dragon breathes lightning in a @Template[type:line|distance:80]{80-foot line} that deals [[/r 8d12[electricity]]] damage (@Check[type:reflex|dc:33|basic:true|traits:arcane,electricity,evocation|overrideTraits:true] save).</li>\n<li><strong>Repulsion Gas</strong> The dragon breathes a @Template[type:line|distance:80]{80-foot line} of repulsive gas. Each creature in the area must succeed at a @Check[type:will|dc:33|traits:arcane,abjuration,incapacitation,mental|overrideTraits:true] save or become @UUID[Compendium.pf2e.conditionitems.Fleeing]{Fleeing} from the dragon for 1 round (or 2 rounds on a critical failure).</li>\n</ul>"
```
We want these parameters in the order listed in the style guide for consistency.